shell/pages/prefs.vue

```diff
@@ -201,17 +201,21 @@ export default {
       </div>
     </div>
 
+    <hr />
+    <div class="row">
+      <div class="col span-3 prefs-advanced">
```

Comment:
`span-3` causes some extreme shrinkage and can be removed

![image](https://user-images.githubusercontent.com/18697775/170711875-cd02898f-c4a6-43fb-b1b9-2743a62de69a.png)